contrib/vagrant/provision-util.sh

```diff
@@ -14,7 +14,7 @@ os::provision::build-origin() {
 
   # This optimization is intended for devcluster use so hard-coding the
   # arch in the path should be ok.
-  if [[ -f "${origin_root}/_output/local/bin/linux/amd64/oc" ]] &&
+  if [[ -f "$(os::build::find-binary oc "${origin_root}")" ]] &&
```

Comment:
I think `find-binary` doesn't need a second argument, just the name of the binary to find.